telluric/collections.py

```diff
@@ -78,12 +100,19 @@ def convex_hull(self):  # type: () -> GeoVector
             shapes = []
 
         return GeoVector(
-            cascaded_union([sh for sh in shapes if sh.is_valid]).convex_hull,
+            cascaded_union([sh for sh in shapes if sh.is_valid]).simplify(0),
```

Comment:
Sometimes `buffer(0)` is used for fixing "bowties". But what is the purpose of `simplify(0)` here?

Reply:
In this case, `simplify(0)` "joins" straight edges, so it turns (0, 1, 2) into (0, 2).